Autopsy of first build.

Purpose: Find out why it didn't fly (Identifty the problem so we know what to solve for in the next build)

# Test flight

<img src="img/flying00.png" alt="Pic here" style="width:400px">
So the canard is at a negative angle of attack.
<img src="img/IMG_3100.JPG" alt="Pic here" style="width:400px">
Results: Inherently unstable.
<img src="img/IMG_3101.JPG" alt="Pic here" style="width:400px">
Results: =D

I believe this only works if the Lift (not CL) slope of the wing is stepper than the canard's.

# Build description

<img src="img/IMG_3102.JPG" alt="Pic here" style="width:800px">
<img src="img/IMG_3103.JPG" alt="Pic here" style="width:400px">
<img src="img/IMG_3104.JPG" alt="Pic here" style="width:400px">
<img src="img/IMG_3105.JPG" alt="Pic here" style="width:400px">
<img src="img/IMG_3106.JPG" alt="Pic here" style="width:400px">
<img src="img/IMG_3107.JPG" alt="Pic here" style="width:800px">
<img src="img/IMG_3109.JPG" alt="Pic here" style="width:400px">
<img src="img/IMG_3110.JPG" alt="Pic here" style="width:800px">

# Thrust test

<img src="img/thrust.png" alt="Pic here" style="width:700px">

I measured the thrust against a scale of battery #1, which is charged at 8.15V (Is this the same one we used for flying yesterday?)

In [4]:
g = 9.8
zero = 630 # grams
read1 = 641 # quarter throttle
read2 = 744 # half throttle
read3 = 830 # three-quarter throttle
read4 = 872 # full throttle

label = ["quarter", "half", "three-quarter", "full"]
for (i,read) in enumerate([read1, read2, read3, read4])
    val = (read-zero)/1000*g
    println("$(round(val,2)) N at $(label[i]) throttle")
end

0.11 N at quarter throttle
1.12 N at half throttle
1.96 N at three-quarter throttle
2.37 N at full throttle


Here is a second test:

In [1]:
g = 9.8
zero = 630 # grams
read1 = 682 # quarter throttle
read2 = 740 # half throttle
read3 = 790 # three-quarter throttle
read4 = 875 # full throttle

label = ["quarter", "half", "three-quarter", "full"]
for (i,read) in enumerate([read1, read2, read3, read4])
    val = (read-zero)/1000*g
    println("$(round(val,2)) N at $(label[i]) throttle")
end

0.51 N at quarter throttle
1.08 N at half throttle
1.57 N at three-quarter throttle
2.4 N at full throttle


Next we opened the box on both ends and in an attempt to reduce obstruction
<img src="img/test02.png" alt="Pic here" style="width:700px">

In [2]:
g = 9.8
zero = 268 # grams
read1 = 305 # quarter throttle
read2 = 395 # half throttle
read3 = 445 # three-quarter throttle
read4 = 570 # full throttle

label = ["quarter", "half", "three-quarter", "full"]
for (i,read) in enumerate([read1, read2, read3, read4])
    val = (read-zero)/1000*g
    println("$(round(val,2)) N at $(label[i]) throttle")
end

0.36 N at quarter throttle
1.24 N at half throttle
1.73 N at three-quarter throttle
2.96 N at full throttle


That's an increase of about 0.6 N on full throttle

XFLR5 shows that our CD during take-off is about 0.065, resulting in the following drag:

In [28]:
CD = 0.065
Sref = 0.23329
V = 8.5
rho = 1.1
D = CD*(0.5*rho*V^2*Sref)

println("Drag at $V m/s: $(round(D,2)) (N)")

Drag at 8.5 m/s: 0.6 (N)


Hence, our motors should be enough to get to climb

# Structural failure

<img src="img/failure00.JPG" alt="Pic here" style="width:700px">
<img src="img/failure01.JPG" alt="Pic here" style="width:700px">
<img src="img/IMG_3096.JPG" alt="Pic here" style="width:700px">
<img src="img/failure02.JPG" alt="Pic here" style="width:700px">

1. **Crack:** This is a good failure, it cracked without falling apart.
2. **Critical failure:** Canard's twin boom ripped off the wing.
3. **Crack:** All good.
4. **Critical failure:** Winglet ripped off tip of wing.
5. **Crack:** Tip of wing cracked and turned up. Did we fly like this in the last test?
6. **Crack:** Attachment of canard to twin boom ripped of along the the tips of the canard. I think this is better than having the canard breaking in half.

**SOLVE FOR**
* 2) Find a more structurally-sound way of attaching the twin boom. Would it be better to attach it to the fuselage instead of the wing? Also, the videos show the canard fluttering.
* 4) Mount the winglet to the wing's spar instead of just the foam.

## Predicting for failure

Let's go through the design process again with the exact build we fly and see if at any point our analysis should have pointed out that it was going to be unstable.

**MODELING MISTAKES**
* Wing span: We designed for 0.9 m in our analyses, but the actual build was 1.2 m.
* Wing sweep: We designed for 30deg, but the actual build is 20deg.
* Wing AOA: We designed for mounting the wing at an angle of attack of -5deg, but I believe we actually mounted at 0deg.
* Canard chord: We designed for a chord of 10cm, but the aileron made it an actual chord of 12cm.
* Canard twin-boom: We designed for a distance of 43.5cm from LE of main wing to LE of canard, but the actual build was 47cm.
* Canard AOA: We designed for mounting the canard at an angle of attack of 0.8deg, but I believe we actually mounted at 0deg, and in actual flight the twin boom deflect to at least -8deg (see pic below).

<img src="img/flying00.png" alt="Pic here" style="width:400px">

**FLIGHT CONDITIONS**
* Since the CG was way back, we put a 68g weight on the canard to move it bring the CG close to the wing's nose. I calculated that our weight during flight ended up being 680g.

I set up the wing at a negative angle of attack such that in addition to the root's twist the effective AOA of the wing's root is 0.

With the canard at no angle of attack I get this:
<img src="img/c0.png" alt="Pic here" style="width:1000px">
<img src="img/c0curves.png" alt="Pic here" style="width:1000px">
looking at the pitching moment, this should have been stable. However, it seems that it would be stable at an AOA of about 10degs. Definitely the plane would stall before that.

With the canard at a negative angle of attack of -5deg I get this:
<img src="img/cneg5.png" alt="Pic here" style="width:1000px">
<img src="img/cneg5curves.png" alt="Pic here" style="width:1000px">
which the pitching moment curve ends up being unstable. If we actually flew with the twin boom deflected down, it is possible we flew on this unstable configuration for the last flight.

## Redesign

### REDESIGN #1

If we shorten the twin boom (move the canard closer to the wing to a distance of 0.37m from LE to LE), the slope of Cm is more negative, and placing the canard at 2degs AOA the pitching moment stabilizes at about 2degs:

<img src="img/c2shorter.png" alt="Pic here" style="width:1000px">
<img src="img/c2shortercurves.png" alt="Pic here" style="width:1000px">

However, at 5m/s at AOA=2degs we only have a CL of 0.54, which gives the following lift:

In [11]:
M = (300 + 306 + 68)/1000
W = M*9.8
CL = 0.54
rho = 1.1
V = 5
Sref = 0.23329
L = CL * (0.5*rho*V^2*Sref)
println("Lift at $V (m/s): $(round(L,2)) (N)")
println("Lift needed: $(round(W, 2)) \t CL needed at $V (m/s): $(round( W/(0.5*rho*V^2*Sref) , 2 ))")

Lift at 5 (m/s): 1.73 (N)
Lift needed: 6.61 	 CL needed at 5 (m/s): 2.06


Which a CL of 2.06 requires an AOA of about 20... imposible. Lets calculate stall speed then.

In [14]:
M = (300 + 306 + 68)/1000
W = M*9.8
rho = 1.1
Sref = 0.23329
L = W
CL = 0.54
V = sqrt(L/CL / (0.5*rho*Sref))
println("V required for Lift=$(round(W,2)) (N) : $(round(V,2)) (m/s)")

V required for Lift=6.61 (N) : 9.76 (m/s)


That seems way high.

### REDESIGN #2

I set the canard at AOA=1deg and move it to a 0.42m from the wing

<img src="img/c2short420.png" alt="Pic here" style="width:1000px">
<img src="img/c2short420curves.png" alt="Pic here" style="width:1000px">

However, at 5m/s stabilizes at AOA=5.3 (very close to stall), but a CL of 0.786, requiring the following speed for take off:

In [16]:
M = (300 + 306 + 68)/1000
W = M*9.8
rho = 1.1
Sref = 0.23329
L = W
CL = 0.786
V = sqrt(L/CL / (0.5*rho*Sref))
println("V required for Lift=$(round(W,2)) (N) : $(round(V,2)) (m/s)")

V required for Lift=6.61 (N) : 8.09 (m/s)


### REDESIGN #3

I increased both tip and root chord on the main by 5cm, placed the canard 0.58m from the wing at AOA=1, and I was able to increase CL a tiny little bit:

<img src="img/bigwing00.png" alt="Pic here" style="width:1000px">
<img src="img/bigwing00curves.png" alt="Pic here" style="width:1000px">

Stabilizes at AOA=5.3 with a CL of about 0.9, requiring the following speed for take off:

In [20]:
M = (300 + 306+60)/1000
W = M*9.8
rho = 1.1
Sref = 0.23329
L = W
CL = 0.9
V = sqrt(L/CL / (0.5*rho*Sref))
println("V required for Lift=$(round(W,2)) (N) : $(round(V,2)) (m/s)")

V required for Lift=6.53 (N) : 7.52 (m/s)


### REDESIGN #4

Then increasing the span by 20cms, and placing the canard 0.68m from the wing at AOA=1:

<img src="img/bigwing01.png" alt="Pic here" style="width:1000px">
<img src="img/bigwing01curves.png" alt="Pic here" style="width:1000px">

Stabilizes at AOA=5.9 with a CL of about 1.1, requiring the following speed for take off:

In [21]:
M = (300 + 306+60)/1000
W = M*9.8
rho = 1.1
Sref = 0.23329
L = W
CL = 1.1
V = sqrt(L/CL / (0.5*rho*Sref))
println("V required for Lift=$(round(W,2)) (N) : $(round(V,2)) (m/s)")

V required for Lift=6.53 (N) : 6.8 (m/s)


Ok, increasing the wing size we can lower the stall speed, but how much is our weight increasing?

# TAKE-AWAYS

* It seems that the distance from the canard to the CG is the most sensitive parameter for stability, along with the AOA of the canard. We definitely flew with an unstable configuration (either an AOA for Cm stabilization that stalls the plane, or a unstable Cm curve with the canard deflected down). We need to have a setup of at least REDESIGN #1, but with our current wing the minimum take-off speed we can get is about 8.5 m/s.
* By increasing either span or chord of the main wing (REDESIGNS #3 and #4) the take-off speed can go as low as 6.8 m/s, but we still have to include in the calculation the increased weight for the bigger wing.
* Rough estimates of drag at 8 m/s are around 0.6 N, and motor static tests show they provide a thrust of 2.37 N  (considering fuselage obstruction). I think we have enough power to climb with ease.
* Before every test, make sure all control surface are responding well.
* Mount the winglet to the wing's spar instead of just the foam.
* Find a more structurally-sound way of attaching the twin boom. Would it be better to attach it to the fuselage instead of the wing? Also, the videos show the canard fluttering.